syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -431,6 +431,41 @@ def addmm(bias, input_tensor, weight):
 
         module.addmm = addmm
 
+        def sigmoid(tensor):
+            """
+            Overloads torch.sigmoid to be able to use MPC
+            Approximation with polynomial interpolation of degree 10 over [-10,10]
+            Ref: https://mortendahl.github.io/2017/04/17/private-deep-learning-with-mpc/#approximating-sigmoid
+            """
+
+            weights = [0.5, 0.216578258, -0.0083312848, 0.0001876528, -1.9669e-06, 7.5e-09]
+            degrees = [0, 1, 3, 5, 7, 9]
+
+            max_degree = 9
+            max_idx = degrees.index(max_degree)
+            selected_weights = weights[1:max_idx]
```

Comment:
I don't think these two variables (selected_weights, selected_degrees) are needed. You can just use weights[1:max_idx] and degrees[1:max_idx] in the for loop.

Reply:
OK